samples/notebooks/dotnet/07-DALL-E-2.ipynb

```diff
@@ -80,11 +80,22 @@
     "var (useAzureOpenAI, model, azureEndpoint, apiKey, orgId) = Settings.LoadFromFile();\n",
     "\n",
     "// Configure the three AI features: text embedding (using Ada), text completion (using DaVinci 3), image generation (DALL-E 2)\n",
-    "var kernel = new KernelBuilder()\n",
-    "    .WithOpenAITextEmbeddingGenerationService(\"text-embedding-ada-002\", apiKey)\n",
-    "    .WithOpenAITextCompletionService(\"text-davinci-003\", apiKey, orgId)\n",
-    "    .WithOpenAIImageGenerationService(apiKey, orgId)\n",
-    "    .Build();\n",
+    "var builder = new KernelBuilder();\n",
+    "\n",
+    "if(useAzureOpenAI)\n",
+    "{\n",
+    "    builder.WithAzureTextEmbeddingGenerationService(\"text-embedding-ada-002\", azureEndpoint, apiKey, orgId);\n",
+    "    builder.WithAzureTextCompletionService(\"text-davinci-003\", azureEndpoint, apiKey, orgId);\n",
```

Comment:
The 'orgId' param doesn't exist on the "WithAzure..." calls as this is a (non-Azure) OpenAI parameter only.

```suggestion
    "    builder.WithAzureTextEmbeddingGenerationService(\"text-embedding-ada-002\", azureEndpoint, apiKey);\n",
    "    builder.WithAzureTextCompletionService(\"text-davinci-003\", azureEndpoint, apiKey);\n",
```